### ISSUE TITLE
How to get the grad of torch_sparse.SparseTensor

### ISSUE THREAD

**Issue by Jiayi-Pan** [rusty1s/pytorch_sparse#186]
I wonder if there's any methods/variables does the same job as torch.Tensor.grad in torch_sparse.SparseTensor? (it does not have a grad attribute

**Comment by rusty1s** [comment]
Gradient computation can be enabled for the `value` of a `SparseTensor`, e.g.:
```python
row = ...
col = ...
value = torch.randn(row.size(0).requires_grad()

mat = SparseTensor(row=row, col=col, value=value)
```

**Comment by Jiayi-Pan** [comment]
Thank you! I've solved it by using ```SparseTensor.storage._value.grad```
